components/automate-ui/src/app/components/input/input.directive.scss

```diff
@@ -29,6 +30,7 @@
   &:focus,
   &:active {
     border-color: $chef-primary-bright;
+    outline: none;
```

Comment:
we defined our own version of a focused input outline, so this outline needs to be removed - uniform across all inputs.